yt_dlp/YoutubeDL.py

```diff
@@ -1408,9 +1314,11 @@ def check_filter():
                     ret = None if incomplete else match_filter(info_dict)
                 if ret is NO_DEFAULT:
                     while True:
-                        filename = self._format_screen(self.prepare_filename(info_dict), self.Styles.FILENAME)
-                        reply = input(self._format_screen(
-                            f'Download "{filename}"? (Y/n): ', self.Styles.EMPHASIS)).lower().strip()
+                        filename = self.logger.format(
+                            LogLevel.INFO, self.prepare_filename(info_dict), Style.FILENAME)
+                        query = self.logger.format(
+                            LogLevel.INFO, f'Download "{filename}"? (Y/n): ', Style.EMPHASIS)
+                        reply = input(query).lower().strip()
```

Comment:
Should this not use `self.logger.screen` first and then a plain input? That way we can know for sure which output will be used and if we can color it.

Reply:
Instead, im thinking it might makes more sense to provide a `Console.input` method instead. That way custom `Console` implementations could use that more effectively as well.